### PR TITLE
Slack huddles support

### PR DESCRIPTION
**Pull request by aperez-worklytics**
Adding rules for `Room` object as part of the response of *conversation history* endpoint for Slack.

### Features
[Huddle psoxy support](https://app.asana.com/0/0/1207877994208971)

### Change implications

 - dependencies added/changed? **no**
 - something important to note in future release notes?
   - NOTE in `CHANGELOG.md` anything that will show up in `terraform plan`/`apply` that isn't
     obviously a no-op? **no**
   - breaking changes? **no** if in module/example that is NOT marked `alpha`, requires major version
     change

### CODE REVIEW

File: docs/sources/slack/example-api-responses/original/discovery-conversations-history.json [hunk 1]
```diff
@@ -196,7 +196,107 @@
...
+      "user": "USLACKBOT",
+      "channel": "D075J2RKK4Y",
+      "room": {
+        "id": "R07BNSQTZSP",
+        "name": "huddle test topic",
+        "media_server": "",
+        "created_by": "U02K5LRARED",
+        "date_start": 1720542372,
+        "date_end": 1720542756,
+        "participants": [],
```
Comment (aperez-worklytics) [inline docs/sources/slack/example-api-responses/original/discovery-conversations-history.json:210]: Redacted; no idea if it contains the plain list of ids like `participant_history` or the user objects like `parcitipants_events`

File: docs/sources/slack/example-api-responses/original/discovery-conversations-history.json [hunk 2]
```diff
@@ -196,7 +196,107 @@
...
+        "participant_history": [
+          "U02K5LRARED",
+          "U074XMEDGUU"
+        ],
+        "participants_events": {
+          "U02K5LRARED": {
```
Comment (aperez-worklytics) [inline docs/sources/slack/example-api-responses/original/discovery-conversations-history.json:216]: Redacted as currently we don't have support for transforming keys, like *U02K5LRARED* to its pseudonymized identity.
Also, with `participants_history` is enough for the Worklytics app side

File: docs/sources/slack/example-api-responses/original/discovery-conversations-history.json [hunk 3]
```diff
@@ -196,7 +196,107 @@
...
+            "camera_on": false,
+            "camera_off": false,
+            "screenshare_on": false,
+            "screenshare_off": false
+          }
+        },
+        "participants_camera_on": [],
+        "participants_camera_off": [],
+        "participants_screenshare_on": [],
+        "participants_screenshare_off": [],
```
Comment (aperez-worklytics) [inline docs/sources/slack/example-api-responses/original/discovery-conversations-history.json:244]: Sames as `participant_events`

File: docs/sources/slack/example-api-responses/original/discovery-conversations-history.json [hunk 4]
```diff
@@ -196,7 +196,107 @@
...
+        "is_prewarmed": false,
+        "is_scheduled": false,
+        "attached_file_ids": [],
+        "media_backend_type": "free_willy",
+        "display_id": "",
+        "external_unique_id": "b88ef6b8-9450-4633-83a2-dfb2cfd12713",
+        "app_id": "A00",
+        "call_family": "huddle",
+        "pending_invitees": {},
+        "last_invite_status_by_user": {
```
Comment (aperez-worklytics) [inline docs/sources/slack/example-api-responses/original/discovery-conversations-history.json:266]: Sames as `participant_events`

**Review by eschultink — APPROVED** [review]

**Pull request merged**